docs/conf.py

```diff
@@ -31,8 +30,7 @@
 # Add any Sphinx extension module names here, as strings. They can be
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
-extensions = ['sphinx.ext.autodoc',
-    'sphinx.ext.ifconfig','sphinx.ext.todo']
+extensions = []
```

Comment:
Not sure if this is not a step backwards

Reply:
I used the standard Read The Docs template. 
I think the misunderstanding here is that so far docs have been used to document the code, while here we have the documentation for the product.

Here is the important topic: If we want to maintain code documentation in /docs, then we must move the product documentation in a separate folder or even repository. We must make this decision asap.